moabb/datasets/liu2024.py

```diff
@@ -0,0 +1,310 @@
+"""Liu2024 Motor imagery dataset."""
+
+import os
+import shutil
+import warnings
+import zipfile as z
+from pathlib import Path
+from typing import Any, Dict, Tuple
+
+import mne
+import numpy as np
+import pandas as pd
+from mne.channels import read_custom_montage
+
+from moabb.datasets import download as dl
+from moabb.datasets.base import BaseDataset
+
+
+# Link to the raw data
+LIU2024_URL = "https://figshare.com/ndownloader/files/38516654"
+
+# Links to the electrodes and events information files
+LIU2024_ELECTRODES = "https://figshare.com/ndownloader/files/38516078"
+LIU2024_EVENTS = "https://figshare.com/ndownloader/files/38516084"
+
+
+class Liu2024(BaseDataset):
+    """
+
+    Dataset [1]_ from the study on motor imagery [2]_.
+
+    .. admonition:: Dataset summary
+
+
+        ========= ======== ======= ========== ================= ============ =============== ===========
+        Name      #Subj    #Chan   #Classes   #Trials / class   Trials len   Sampling rate   #Sessions
+        ========= ======== ======= ========== ================= ============ =============== ===========
+        Liu2024   50       29      2          40                4s           500Hz           1
```

Comment:
Same here: why 29 channels in the docstring and 30 in the PR text?

When loading the data of subject 1, I got 30 EEG channels plus HEOL and HEOR

Reply:
It's actually 29 EEG, 1 reference channel CPz, 2 EOG HEOL and HEOR and 1 stim channel. 
I renamed the HEOR channel to VEOR as it's named in their montage .It should be good with the last push.